### ISSUE TITLE
Ftrack Integrator may use wrong entity where publish is integrated

### ISSUE THREAD

**Issue by iLLiCiTiT** [pypeclub/pype#1104]
## Description
Plugin `Integrate Ftrack Api` use `"ftrackEntity"` or `"ftrackTask"` which is collected in plugin `Collect Ftrack Api` but may not be the final entity for integration to ftrack e.g. during batch publishing or editorial.

Found out during PR https://github.com/pypeclub/pype/pull/1101

**Comment by mkolar** [comment]
@iLLiCiTiT I reckon we can close this considering we havent' ancountered it since it since. Or at least we had no report on it.

**Comment by iLLiCiTiT** [comment]
I agree we can close this. It is tightly connected to fact that instance does not have information about asset to which it belongs and fallbacks to context data.
